033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md

```diff
@@ -1,12 +1,12 @@
-# Challenge 4: Offline Cutover and Validation
+# Challenge 3: Offline Cutover and Validation
 
-[< Previous Challenge](./03-offline-migration.md) - **[Home](../README.md)** - [Next Challenge >](./05-online-migration.md)
+[< Previous Challenge](./02-offline-migration.md) - **[Home](../README.md)** - [Next Challenge >](./04-online-migration.md)
 
 ## Introduction
- Reconfigure the application to use the appropriate connection string that uses Azure DB and validate that the application is working
+ Reconfigure the application to use the appropriate connection string that uses Azure DB and validate that the application is working. You have to do this by redeploying the Pizzeria container application(s). 
 
 ## Description
-You will reconfigure the application to use a connection string that points to the Azure DB for PostgreSQL/MySQL. You will need to update the ContosoPizza/values-mysql.yaml or ContosoPizza/values-postgresql.yaml values file with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate Azure DB values for PostgreSQL/MySQL:
+You will reconfigure the application to use a connection string that points to the Azure DB for PostgreSQL/MySQL. You will need to update the ContosoPizza/values-mysql.yaml, ContosoPizza/values-postgresql.yaml and/or ContosoPizza/values-oracle.yaml values file(s) with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate values for Azure DB for PostgreSQL/MySQL:
```

Comment:
`ContosoPizza/values-msql.yaml` use the backticks to format code or file references